src/Core/Components/DateTime/FluentDatePicker.razor

```diff
@@ -1,7 +1,7 @@
 @namespace Microsoft.FluentUI.AspNetCore.Components
 @inherits FluentCalendarBase
 
-<FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" />
+<FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" Required=@Required TValue="DateTime?" />
```

Comment:
What is this `TValue`?

Reply:
Left-over from previos Input paramer thing. Will remove